_layouts/default.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Mocha - the fun, simple, flexible JavaScript test framework</title>
+    <link rel="stylesheet" href="assets/normalize.min.css">
+    <link rel="stylesheet" href="assets/style.css">
+    <link rel="shortcut icon" href="favicon.ico">
+    <script src="assets/highlight.js"></script>
+    <script>
+      setTimeout(function(){
+        document.body.className += ' onload';
+      }, 0);
+    </script>
+    <!--[if lt IE 9]><script src='//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv.min.js'></script><![endif]-->
+  </head>
+  <body>
+    <section id="content">
+      <h1><a href="https://github.com/mochajs/mocha">Mocha</a></h1>
+      <p id="tag"><em>simple</em>, <em>flexible</em>, <em>fun</em></p>
+      {{content}}
+    </section>
+    <footer>
+      <span>© 2011 TJ Holowaychuk. All rights reserved.</span>
```

Comment:
I've left this as is, since this PR is only about restructuring things (not updating content). Might wanna change this.